site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -487,6 +487,28 @@ public function updateAnnouncement() {
         );
     }
 
+
+    /**
+     * @Route("/courses/{_semester}/{_course}/office_hours_queue/stats", methods={"GET"})
+     * @return MultiResponse
+     */
+    public function showQueueStats() {
+        if (!$this->core->getConfig()->isQueueEnabled()) {
+            return MultiResponse::RedirectOnlyResponse(
+                new RedirectResponse($this->core->buildCourseUrl(['home']))
```

Comment:
There should be an error message prompting the user the queue is not enabled here